app/main.py

```diff
@@ -1,8 +1,40 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int, clean_mark: int, brand: str) \
+            -> None:
```

Comment:
Code Style issue: Avoid using the line continuation character (\) in Python. Instead, use parentheses to continue expressions over multiple lines. This will make your code more readable and maintainable.